src/apps/Permissions/AppInstanceLabel.js

```diff
@@ -1,47 +1,41 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import styled from 'styled-components'
-import { Badge, Viewport, breakpoint } from '@aragon/ui'
+import { Badge, useViewport, breakpoint } from '@aragon/ui'
 import { AppType, EthereumAddressType } from '../../prop-types'
 import { shortenAddress } from '../../web3-utils'
 import AppIcon from '../../components/AppIcon/AppIcon'
 
-class AppInstanceLabel extends React.PureComponent {
-  static propTypes = {
-    app: AppType.isRequired,
-    proxyAddress: EthereumAddressType.isRequired,
-    showIcon: PropTypes.bool,
-  }
+const AppInstanceLabel = ({ app, proxyAddress, showIcon = true }) => {
```

Comment:
We should use `React.memo()` to keep it equivalent to `React.PureComponent` :+1: